beam/www/beam/components/Transfer.vue

```diff
@@ -13,6 +13,7 @@ import { ref } from 'vue'
 
 import { useDataStore } from '@/store'
 import type { ListViewItem, StockEntry } from '@/types'
+import ControlButtons from '../components/ControlButtons.vue'
```

Comment:
Minor nitpick, but a nicer way to shorthand this is with TS' path configs, so this can become:

```suggestion
import ControlButtons from '@/components/ControlButtons.vue'
```